Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -253,14 +368,20 @@ extension HTTPClient {
             self.lock = Lock()
         }
 
+        /// `EventLoopFuture` of this request.
```

Comment:
Should this be something like: "`EventLoopFuture` for the response returned by this request"?

Reply:
fixed, thanks!